azure/hostedservice/entities.go

```diff
@@ -1,9 +1,16 @@
-package hostedServiceClient
+package hostedservice
 
 import (
 	"encoding/xml"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/azure"
 )
 
+//HostedServiceClient is used to manage operations on Azure Hosted Services
+type HostedServiceClient struct {
+	client *azure.Client
```

Comment:
you can use type embedding (see Embedding section of Effective Go) if you can have only 1 field like this.